internal/controller/hbase_controller.go

```diff
@@ -41,6 +43,16 @@ type HBaseReconciler struct {
 	GhAdmin gohbase.AdminClient
 }
 
+var (
+	hbaseReconciliationPhaseMetric = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "hbase_reconciliation_phase",
+			Help: `Current phase of HBase (1=ApplyingChanges, 2=Ready, 3=Invalid, 0=Unknown)`,
+		},
+		[]string{"namespace", "name"},
```

Comment:
Would add `phase` to the labels and set the metric value to 1 whenever a phase is active.

E.g.:

```
[]string{"namespace", "name", "phase"},
```

And in updateReconciliationPhaseMetrics write:

```
hbaseReconciliationPhaseMetric.Reset()
hbaseReconciliationPhaseMetric.WithLabelValues(hb.Namespace, hb.Name, hb.Status.Phase).Set(1)
```

This way you don't have to remember which phase maps to which value. To check if we have finished applying changes, make the query `hbase_reconciliation_phase{phase="Ready"} == 1`

Other metrics also follow this pattern, e.g. `kube_pod_status_phase{phase="Pending"}`